ruby-challenges.rb

```diff
@@ -32,5 +65,27 @@
 # Expected output: 'LEARN is not a palindrome'
 palindrome_test_case3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+palindrome_test_case4 = 'asdfGhjkl'
+# Pseudo code:
+
+# method that flips strings to check if its a palindrome
+
+def flip (word)
+# create a variable to hold the original word so that we can alter the string later on
+    original = word
+    nope = word.reverse
+#.reverse to reverse our string
+    word = word.downcase
+#.downcase to downcase our entire given string so that we can compare it to original 
+# create an if else statement to ouput the results 
+    if word == word.reverse
+        return "#{original} is a palindrome"
+    else 
+        return "#{original} is not a palindrome the output is #{nope} "
+    end
+end
```

Comment:
I like your logic.  Just remember Ruby methods don't alter the original data, so you don't need an additional variable.  :)